frame/staking/src/testing_utils.rs

```diff
@@ -59,6 +59,22 @@ pub fn create_funded_user<T: Config>(
 	user
 }
 
+/// Grab a funded user.
+pub fn create_funded_user_b<T: Config>(
+	string: &'static str,
+	n: u32,
+	balance_factor: crate::BalanceOf<T>,
+) -> T::AccountId {
+	use sp_runtime::traits::Bounded;
+	let user = account(string, n, SEED);
+	let balance =
+		(T::Currency::minimum_balance() * balance_factor).max(BalanceOf::<T>::max_value());
+	T::Currency::make_free_balance_be(&user, balance);
+	// ensure T::CurrencyToVote will work correctly.
+	T::Currency::issue(balance); // TODO I don't get this .. will drop NegativeImbalance which cancels itself out
```

Comment:
note

Reply:
does everything work as expected without it?